src/rgw/motr/gc/gc.h

```diff
@@ -23,7 +23,7 @@
 #include <atomic>
 
 const int64_t GC_MAX_SHARDS_PRIME = 65521;
```

Comment:
Change this to 
const int32_t GC_DEFAULT_QUEUES = 1024;

Reply:
`GC_MAX_SHARDS_PRIME` is not a default value @sachinpunadikar, it a value to restrict users from entering an extremely huge value in config fie.
`1024` will be value of `rgw_gc_max_objs` in the config (solution.yaml) right?

Reply:
If rgw_gc_max_objs is defined in config, GC will use this value as max_indices else it will use GC_DEFAULT_QUEUES which is set to 128 as of now